exspy/exspy/test/test_io.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2023 The HyperSpy developers
+#
+# This file is part of HyperSpy.
+#
+# HyperSpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# HyperSpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with HyperSpy. If not, see <https://www.gnu.org/licenses/#GPL>.
+
+from pathlib import Path
+
+import numpy as np
+import pytest
+
+import hyperspy.api as hs
+from hyperspy import __version__ as hs_version
+
+
+PATH = Path(__file__).resolve()
+
+
+
+def test_load_save_filereader_metadata(tmp_path):
```

Comment:
This test should stay in hyperspy, as it is not testing exspy functionality.

Reply:
Done, thanks!